apps/accounts/backends.py

```diff
@@ -1,14 +1,24 @@
-from django.contrib.auth import get_user_model
+
 from django.contrib.auth.backends import ModelBackend
+from django.contrib.auth import get_user_model
+from django.core.exceptions import ValidationError
 
 class OrgBackend(ModelBackend):
-    def authenticate(self, username=None, password=None): # type: ignore figure out overrides with pyright here
+    def authenticate(self, request, username=None, password=None, **kwargs):

```

Comment:
`request` and `**kwargs`are unused arguments.

Reply:
`request` is needed as a argument for authenticate to work as we are using a custom backend. Same goes for `**kwargs`